app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitors must be vaccinated!")
+        if visitor["vaccine"]["expiration_date"] < date.today():

```

Comment:
Using `.get()` is a safer way of getting a value from a dict. You should use `[]` only when you are sure of the structure of the incoming dict. Generally, in production code prefer using `.get()` over `[]`.